.buildkite/pipeline.yml

```diff
@@ -1,6 +1,36 @@
 steps:
-  - commands:
+  - label: ":hammer: running tests"
+    commands:
       - "make ci"
-    name: ":hammer: build"
+        #- "make test" #currently no ct defined, causes build error
+    key: "tests"
+    agents:
+      queue: "erlang"
+
+  - wait
+
+  - label: ":debian: build deb"
+    commands:
+      - "git checkout ${BUILDKITE_TAG}"
+      - "make ci"
+      #- "make test"
+      - "make release"
+      - ".buildkite/scripts/make_deb.sh"
```

Comment:
I've been trying to keep most build/release steps in the Makefile to keep it all co-located. Not too hung up on it but it'd be good to hear your thoughts on what should go where

Reply:
I think anything which requires a secret should live in a bash script. I can put the command to build a package into the makefile but I have seen a lot of examples where people do the packaging via fpm in bash scripts as well for convenience of interacting with environment variables